test/e2e/httpproxy/httpproxy_test.go

```diff
@@ -427,23 +427,6 @@ descriptors:
 	})
 })
 
-// httpProxyValid returns true if the proxy has a .status.currentStatus
```

Comment:
seems like some commits from a merge or rebase snuck in here?

Reply:
ah wait maybe nvm, i think i did the same thing in one of my PRs that I'm mixing up: https://github.com/projectcontour/contour/pull/4170/files#diff-85ebf4a94b7ed92e12d4ccea5e41be34ceeaf112df1ebba49de2fec7c297b780R461